### DISCUSSION TITLE
Welcome to RP-Spaceoids Discussions!

### DISCUSSION THREAD

**Discussion by CorpHackRyan** [CorpHackRyan/RP-Spaceoids#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CorpHackRyan** [comment]
My name is Ryan, I'm a long time hobbyist coder. I've tinkered with PCs for as long as I can remember, and just much with software. My background was in the classic warez days of the 90's, BBS's, old DOS games. Funnily enough, I' used to be a big gamer but the recent 20 years or so with the exception of a Gears of War binge, I've been mostly absent from playing games except for chess and poker.

**Reply by patpragman** [reply]
Sup!

**Reply by CorpHackRyan** [reply]
Nice to meet you Pat!

**Reply by patpragman** [reply]
Hah!  We can upvote... obviously this should be horrifically toxic for corporate software operations.  Perfect.

**Reply by CorpHackRyan** [reply]
Terrific!

**Reply by CorpHackRyan** [reply]
> Hah! We can upvote... obviously this should be horrifically toxic for corporate software operations. Perfect.

I like how it has a forum-esque style built in.

**Comment by patpragman** [comment]
My name is Pat.  I'm also a long time hobbyist coder.  I was a professional pilot for 14 years or so but I can't fly anymore, so I'm pivoting to a career in software development.  Anyway I really liked Escape Velocity as a kid, so an asteroids clone seems perfect for a game project!

**Reply by CorpHackRyan** [reply]
You got my upvote.
